src/openforms/forms/api/viewsets.py

```diff
@@ -613,18 +625,42 @@ def logic_rules_list(self, request, *args, **kwargs):
         summary=_("JSON schema"),
         description=_("Generate the JSON schema for a form."),
         parameters=[UUID_OR_SLUG_PARAMETER],
-        request=None,
-        responses={status.HTTP_200_OK: OpenApiTypes.OBJECT},
+        request=FormJsonSchemaOptionsSerializer,
+        responses={
+            status.HTTP_200_OK: OpenApiTypes.OBJECT,
+            status.HTTP_400_BAD_REQUEST: ExceptionSerializer,
+        },
+    )
+    @action(
+        detail=True,
+        methods=["get"],
+        permission_classes=(permissions.IsAdminUser,),
+        renderer_classes=(FormJSONSchemaRenderer,),
     )
-    @action(detail=True, methods=["get"], permission_classes=(permissions.IsAdminUser,))
     def json_schema(self, request, *args, **kwargs):
+        serializer = FormJsonSchemaOptionsSerializer(data=request.GET)
```

Comment:
in API endoints, you can use `request.query_params` with DRF, which works for both GET and POST requests :)

```suggestion
        serializer = FormJsonSchemaOptionsSerializer(data=request.query_params)
```